ferrocene/doc/user-manual/src/targets/index.rst

```diff
@@ -5,7 +5,7 @@ Compilation targets overview
 ============================
 
 Ferrocene has support for multiple compilation targets and host platforms.
-Targets are categorized as either "supported" or "experimental" depending on
+Targets are categorized as either "supported" or "untested" depending on
```

Comment:
This is missing "experimental".

Reply:
Also "qualified" and "quality managed", oops!